README.md

```diff
@@ -34,10 +34,10 @@ jobs:
 
 ## Workflow options
 
-Change these options in the workflow `.yml` file to meet your GitHub project needs.
+Change these options in the workflow `.yml` file to meet your GitHub project needs:
 
-| Setting            | Description                                      | Default value                         |
-| ------------------ | ------------------------------------------------ | ------------------------------------- |
-| `api-token`        | An account API token                             | `${{ secrets.CODACY_API_TOKEN }}`     |
-| `project-token`    | The project API token                            | `${{ secrets.CODACY_PROJECT_TOKEN }}` |
-| `coverage-reports` | Optional Comma separated list of reports to send | `''`                                  |
+| Setting            | Description                                                                            | Default value                         |
```

Comment:
hmm, there are no default values for any of these options
```suggestion
| Setting            | Description                                                                            | Recommended value                         |
```

Reply:
Interesting, while editing the table I was thinking that it was very clever to have those as the default values. :nerd_face: Perhaps it makes sense to do this improvement?

Reply:
true. the way it is right now, your suggestion of using global tokens won't work